testsuite/tests/tool-ocamlc-stop-after/stop_after_scheduling.compilers.reference

```diff
@@ -0,0 +1 @@
+wrong argument 'scheduling'; option '-stop-after' expects one of: parsing typing.
```

Comment:
Is this really the result expected for this test?

Reply:
It's a test for ocamlc, not ocamlopt, so the tool does not support `scheduling` option (unless I messed up the test setup)

Reply:
Ah yes, I missed this point, sorry.